deepchem/models/tensorflow_models/lr.py

```diff
@@ -172,8 +173,10 @@ def predict_proba_on_batch(self, X):
         # transfer 2D prediction tensor to 2D x n_classes(=2) 
         complimentary = np.ones(np.shape(batch_outputs))
         complimentary = complimentary - batch_outputs
-        batch_outputs = np.concatenate(
-            [complimentary, batch_outputs], axis=batch_outputs.ndim - 1)
+        batch_outputs = np.squeeze(
+            np.stack(arrays=[complimentary, batch_outputs], 
+                     axis=batch_outputs.ndim - 1))
+
```

Comment:
Would you mind recovering to the original concatenate function? The squeeze function here may cause problems when the task dimension is only 1(single task dataset like HIV)